app/src/main/java/ch/epfl/sweng/thegramginers/Project/ProjectDetail/ProjectDetailFragment.java

```
@@ -65,6 +65,11 @@ public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
         duration = getView().findViewById(R.id.contractDurationValue);
         date = getView().findViewById(R.id.datesValue);
 
+        if(project.getOwnerId().equals(User.getCurrentUser().getId())){
```

I ran the tests that fail on travis locally and I could notice that you do not init the mock database system at the beginning of Project Description Test. I put some initialization and it gave me the following error : 

```
java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.String.equals(java.lang.Object)' on a null object reference
at ch.epfl.sweng.thegramginers.Project.ProjectDetail.ProjectDetailFragment.onViewCreated(ProjectDetailFragment.java:68)
```

And it turns out that in your test, the project you handle has no owner. So perhaps the bug may come from there. Of course, this is a supposition and just a hint 😊🤞🏻

---

My bad, I tried to fix it as I said but did not notice the owner was already on the constructor. Sorry about that!

---

I fixed that part but there is another problem, in TeamDetailTest, the test team isn't showing at all